readme.md

```diff
@@ -1,44 +1,61 @@
 # remark-smartypants
 
+[![v](https://badgen.net/npm/v/remark-smartypants)][npm]
+[![dt](https://badgen.net/npm/dt/remark-smartypants)][npm]
+
 [remark] plugin to implement [SmartyPants]. Now with 100% more ESM!
 
+## Installing
+
 ```sh
+# using npm
 npm install remark-smartypants
+
+# using yarn
+yarn add remark-smartypants
 ```
 
+## Usage
+
+Example using [remark]:
+
 ```js
-import remark from 'remark'
-import smartypants from 'remakr-smartypants'
+import remark from "remark";
+import smartypants from "remark-smartypants";
```

Comment:
It seems like you have a Prettier extension that's formatting this, could you either turn it off or set it to single quotes and no semicolons?